app/views/sessions/mocklogin.html.erb

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
```

Comment:
Do we need this file?

Reply:
Yes, it's the styling for the log in page which I spent yesterday doing.